gradle.properties

```diff
@@ -1,4 +1,4 @@
-version=0.7.0-alpha.3
+version=special-test
```

Comment:
revert this please

Reply:
Still in progress this PR, let me update it to draft

Reply:
It's missing tests and examples so I can't merge it. But I will remove that marker.

Reply:
OK, change it to draft 👍🏽